examples/tutorials/AlphafoldXDeepChem.ipynb

```diff
@@ -0,0 +1,1626 @@
+{
```

Comment:
<p>Break this down into a few cells and add description between them</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>